File: test_gemini.py
import urllib.request

url = "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-pro-latest:streamGenerateContent?key=invalid"
try:
    req = urllib.request.Request(url, method="POST")
    urllib.request.urlopen(req)
except Exception as e:
    print(e)

File: test_gemini.go
package main

import (
	"fmt"
	"net/http"
)

func main() {
	url := "https://generativelanguage.googleapis.com/v1beta/models/gemini-1.5-pro-latest:streamGenerateContent?key=invalid"
	req, _ := http.NewRequest("POST", url, nil)
	resp, err := http.DefaultClient.Do(req)
	if err != nil {
		fmt.Println("Error:", err)
		return
	}
	defer resp.Body.Close()
	fmt.Println("Status:", resp.Status)
}
